tests/e2e/test_operator.py

```diff
@@ -2142,6 +2143,77 @@ def test_032(self):
     kubectl.delete_chi(chi)
     kubectl.launch(f"delete pod clickhouse-test-032-client", ns=kubectl.namespace, timeout=600)
 
+
+@TestScenario
+@Name("test_033. Check HTTPS support for health check")
+def test_033(self):
+    def check_monitoring_metrics_failure(operator_namespace, operator_pod, expect_pattern, max_retries=10):
+        with Then(f"metrics-exporter /metrics endpoint result should contain {expect_pattern}"):
+            for i in range(1, max_retries):
+                url_cmd = util.make_http_get_request("127.0.0.1", "8888", "/metrics")
+                out = kubectl.launch(
+                    f"exec {operator_pod} -c metrics-exporter -- {url_cmd}",
+                    ns=operator_namespace
+                )
+                
+                rx = re.compile(expect_pattern,re.MULTILINE)
+                matches = rx.findall(out)
+                print(matches)
+                expected_pattern_found = False
+                if matches:
+                    expected_pattern_found = True
+
+                if expected_pattern_found:
+                    break
+                with Then("Not ready. Wait for " + str(i * 5) + " seconds"):
+                    time.sleep(i * 5)
+            assert expected_pattern_found, error()
+
+    with Given("clickhouse-operator pod exists"):
+        kubectl.wait_field("pods", util.operator_label, ".status.containerStatuses[*].ready", "true,true",
+                           ns=settings.operator_namespace)
+        assert kubectl.get_count("pod", ns='--all-namespaces', label=util.operator_label) > 0, error()
+
+    with When("create the chi"):
+        manifest = "manifests/chi/test-033-https-check.yaml"
+        chi = yaml_manifest.get_chi_name(util.get_full_path(manifest))
+
+        kubectl.create_and_check(
+            manifest=manifest,
+            check={
+                "apply_templates": {
+                    settings.clickhouse_template,
+                    "manifests/chit/tpl-persistent-volume-100Mi.yaml",
+                },
+                "object_counts": {
+                    "statefulset": 1,
+                    "pod": 1,
+                    "service": 2,
+                },
+                "do_not_delete": 1,
+            },
+            timeout=600,
+        )
+        
+        kubectl.wait_jsonpath("pod", "chi-test-033-https-check-default-0-0-0", "{.status.containerStatuses[0].ready}", "true",
+                            ns=kubectl.namespace)
+
+    with And(f"apply ClickHouseOperatorConfiguration {settings.chopconf_test_033}"):
+        kubectl.apply(util.get_full_path(settings.chopconf_test_033, lookup_in_host=False), operator_namespace)
```

Comment:
why, you need to store chopconf manifest name in settings?
i think local variable will look more mental here
and i dont 'see settings.py in PR it will not work

Reply:
Moved the file changes to local variable